bl-plugins/api/plugin.php

```diff
@@ -156,6 +156,8 @@ public function beforeAll()
                 $key = $parmC.'/'.$parmD;
             }
             $data = $this->getFiles($key);
+            //$page = new Page($pageKey);
```

Comment:
Why add lines commented?

Reply:
My fault, it was a test and I forgot to remove these two lines before pushing